web-ng/src/app/components/form-field-editor/form-field-editor.component.html

```diff
@@ -75,8 +75,9 @@
       Add option
     </button>
   </div>
+  <hr class="horizontal-line">
   <div fxLayout fxLayoutAlign="end" class="question-actions">
-    <button mat-icon-button type="button" (click)="onFieldDelete()">
+    <button mat-icon-button class="delete-btn" (click)="onFieldDelete()">
```

Comment:
Is this class actually needed? I believe the theme foreground color is the default for icon buttons:
https://material.angular.io/components/button/overview